addons/web/views/report_templates.xml

```diff
@@ -155,6 +155,7 @@
                         <t t-raw="information_block"/>
                     </div>
                 </t>
+                <t t-set="colclass" t-value="colclass + ' recipient_address'"/>
```

Comment:
As recipient_adress doesn't exists in web, I think is not a really good idea to set it in this module. Could you try create an xpath in the snailmail module the will inherit the "address_layout" template and add the class 'recipient_address'